pom.xml

```diff
@@ -401,7 +401,7 @@
         <version.antlr>4.13.0</version.antlr>
         <version.com.carrotsearch.hppc>0.8.1</version.com.carrotsearch.hppc>
         <version.com.fasterxml.classmate>1.5.1</version.com.fasterxml.classmate>
-        <version.com.fasterxml.jackson>2.17.0</version.com.fasterxml.jackson>
+        <version.com.fasterxml.jackson>2.18.0</version.com.fasterxml.jackson>
```

Comment:
If we go with 2.18, we should probably use 2.18.2 that was just released https://github.com/FasterXML/jackson/wiki/Jackson-Release-2.18.2.

Reply:
Definitely. Otherwise within a day or two we'll just end up with a dependabot PR for everyone to approve.